### ISSUE TITLE
Type 'timestamp' is incompatible to type 'timestamptz'

### ISSUE THREAD

**Issue by Alex101499** [guilhermenoronha/parquet-to-hyper#15]
Hi @guilhermenoronha, I'm having this issue due to a column type because the date column in one of my datasets comes with the timestamptz instead timestamp. How can I handle this?

**Comment by guilhermenoronha** [comment]
Could you show me the error message and the column format?  Maybe I need to fix that addind another column type in the _convert_struct_field function to handle the timestampz you mentioned, but I need to be sure and test it.

Sincerely

**Comment by Alex101499** [comment]
Yes sure. This is the complete error, prueba.snappy.parquet is the parquet file name and date is the column name:
WARNING:root:File prueba.snappy.parquet could not be processed. Type 'timestamp' is incompatible to type 'timestamptz' of column 'date' in Parquet file.: Parquet file ./data\prueba.snappy.parquet
